crates/lib/src/bootc_composefs/repo.rs

```diff
@@ -54,6 +54,25 @@ pub(crate) async fn initialize_composefs_repository(
     .await
 }
 
+/// skopeo (in composefs-rs) doesn't understand "registry:"
+/// This function will convert it to "docker://" and return the image ref
+///
+/// Ex
+/// docker://quay.io/some-image
+/// containers-storage:some-image
+pub(crate) fn get_imgref(transport: &String, image: &String) -> String {
```

Comment:
This one looks very unit testable

I haven't followed the flow here but ideally we do https://github.com/youki-dev/oci-spec-rs/issues/205 and then we're always using a parsed representation for these things.